qa/tasks/cephfs/mount.py

```diff
@@ -598,6 +598,32 @@ def stat(self, fs_path, wait=True):
         else:
             return proc
 
+    def lstat(self, fs_path, wait=True):
```

Comment:
To avoid code duplication, it'd be better to add an option to `stat` to do an `lstat` instead.

Reply:
Adding `follow_symlink=False` argument to stat().

Reply:
Should we do the same to `cephfs.pyx` or maintaining a call by the name of `lstat` is more important?

Reply:
Keeping `lstat` as a helper that simply calls `stat` would be valuable I think.

Reply:
Ok.

Reply:
I've made the change and tested it locally.